<launch>
  <include file="$(find line_detection)/launch/test_preproc_sim.launch" />

  <group ns="robot1">
    <node
      name="dbs_line_detector"
      pkg="line_detection"
      type="dbscan_lane_detector.py"
      required="true"
      output="screen"
    >
      <param name="img_in_topic" type="str" value="image_preproc" />
      <param name="desired_twist_out_topic" type="str" value="desired_twist" />
    </node>
  </group>
</launch>
